install.rdf

```diff
@@ -1,34 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Please don't remove this file. It is required for a plugin which supports both Zotero 6 and Zotero 7.